core/src/main/java/cucumber/runtime/model/CucumberFeature.java

```diff
@@ -14,17 +14,19 @@
 import java.io.PrintStream;
 import java.io.Serializable;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class CucumberFeature implements Serializable {
     private static final long serialVersionUID = 1L;
     private final String uri;
     private String language;
     private GherkinDocument gherkinDocument;
     private String gherkinSource;
+    public static final Pattern RERUN_PATH_SPECIFICATION = Pattern.compile("(?m:^| |)(.*?\\.feature(?:(?::\\d+)*))");
```

Comment:
`?m: ` for multi line or this input:

```
/some/path/foo.feature
[space]/some/path/foo.feature
```
Will yield: `["/some/path/foo.feature", "/some/path/bar.feature"]`. 

Not sure if this actually a bad thing though.